apps/crawler/src/scanners/accounts-scanner.service.ts

```diff
@@ -0,0 +1,112 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Account } from '@entities/Account';
+import { SdkService } from '../skd/sdk.service';
+import { normalizeSubstrateAddress, normalizeTimestamp } from '@common/utils';
+import { AllBalances } from '@unique-nft/sdk/types';
+import IScannerService from './scanner.interface';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class AccountsScannerService implements IScannerService {
+  private readonly logger = new Logger(AccountsScannerService.name);
+
+  constructor(
+    @InjectRepository(Account)
+    private accountsRepository: Repository<Account>,
+    private configService: ConfigService,
+    private sdkService: SdkService,
+  ) {}
+
+  async scan() {
+    this.logger.log('Start full scan...');
+
+    const PARALLEL_TASKS = this.configService.get('PARALLEL_TASKS') || 10;
+    const currentBlockNumber = await this.sdkService.getCurrentBlockNumber();
+    const accountsIds = await this.sdkService.getAccountsIds();
+
+    try {
+      while (accountsIds.length > 0) {
+        const accountsIdsForProcessing = accountsIds.splice(0, PARALLEL_TASKS);
+        await Promise.all(
+          accountsIdsForProcessing.map((accountId) =>
+            this.processAccount({
+              accountId,
+              blockNumber: currentBlockNumber,
+            }),
+          ),
+        );
+
+        this.logger.verbose(
+          `Processed ${accountsIdsForProcessing.length} accounts`,
+        );
+      }
+    } catch (err) {
+      this.logger.error(err);
```

Comment:
If we not using Sentry as main logger, we should add Sentry logging for all exceptions manually.

Reply:
Added. There was no santry at the moment of wrighting this code.